devel/getdns/Portfile

```diff
@@ -29,12 +28,17 @@ checksums           rmd160  9f526ba64162308ee963b206e36540101bbe9441 \
                     sha256  f1404ca250f02e37a118aa00cf0ec2cbe11896e060c6d369c6761baea7d55a2c \
                     size    703262
 
-depends_build-append        port:pkgconfig
+depends_build-append        path:bin/pkg-config:pkgconfig
 
 depends_lib                 path:lib/libssl.dylib:openssl \
                             port:libevent \
                             port:libidn2 \
-                            port:libuv
+                            path:lib/pkgconfig/libuv.pc:libuv
+
+# cc1: error: unrecognized command line option "-Wpedantic"
+if {[string match *gcc-4.* ${configure.compiler}]} {
+    patchfiles-append       patch-xcode-gcc.diff
+}
```

Comment:
A patchfile that only takes effect in unusual circumstances is likely not to be updated by future contributors, which can easily result in the port breaking for you and requiring more effort to fix.

Instead of applying this conditionally, how about unconditionally changing the flag from `-Wpedantic` (which newer compilers understand) to `-pedantic` (which older compilers also understand and means the same thing)? You could suggest this change to the developers too.